data/sql/updates/pending_db_world/rev_1629485842027723200.sql

```diff
@@ -1,6 +1,7 @@
 INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629485842027723200');
 
 ALTER TABLE `playercreateinfo_item`
-    CHANGE `amount` `amount` TINYINT(4) SIGNED NOT NULL DEFAULT 1;
+    CHANGE `amount` `amount` INT(11) SIGNED NOT NULL DEFAULT 1;
```

Comment:
```suggestion
    CHANGE `amount` `amount` INT SIGNED NOT NULL DEFAULT 1;
```

Reply:
Also you dont really need anything bigger than SMALLINT

Reply:
@Kitzunu We do, look at the C++ code.